hugegraph-server/hugegraph-api/src/main/java/org/apache/hugegraph/api/filter/AuthenticationFilter.java

```diff
@@ -71,15 +72,15 @@ public class AuthenticationFilter implements ContainerRequestFilter {
 
     private static final Logger LOG = Log.logger(AuthenticationFilter.class);
 
-    private static final List<String> WHITE_API_LIST = ImmutableList.of(
-            "graphs/*/auth/login",
+    private static final AntPathMatcher MATCHER = new AntPathMatcher();
+    private static final Set<String> FIXED_WHITE_API_SET = ImmutableSet.of(
             "versions",
             "openapi.json"
     );
-    private static final AntPathMatcher MATCHER = new AntPathMatcher();
+    // Remove auth/login API from white list
```

Comment:
> we need to make sure the login api is accessible, it's used to get a user token. and also update the comment after checked.

Some context for the modification:
1. When we `unset-context` in AccessLogFilter, the CI will **fail** in `login` when(login) whiteList exist (refer [CI link](https://github.com/apache/incubator-hugegraph/actions/runs/8173840336/job/22347189327?pr=2470))
<img width="1434" alt="image" src="https://github.com/apache/incubator-hugegraph/assets/17706099/a11c895d-56d8-4e10-a0e1-0bb9f604414e">

2. follow the `login` API input param & the validate logic
<img width="942" alt="image" src="https://github.com/apache/incubator-hugegraph/assets/17706099/1cc81ec4-303e-4d5e-b093-08cfa96aa2e5">

Reply:
Because the login method itself will authenticate the user, there is no need to authenticate twice. Of course, it seems there is no problem when authenticating twice.

Reply:
> 1. the CI will **fail** in `login` when(login) whiteList exist (refer [CI link](https://github.com/apache/incubator-hugegraph/actions/runs/8173840336/job/22347189327?pr=2470))

@javeme Yes, but after we fix the TLS leak, the `login/tests` **can't run well** if we only check auth once in LoginAPI, and we'll dig it out the reason, cc @SunnyBoy-WYH mark it as a TODO issue